proto/babylon/finality/v1/query.proto

```diff
@@ -229,11 +231,19 @@ message QuerySigningInfoRequest {
   string fp_btc_pk_hex = 1;
 }
 
+// SigningInfoResponse defines the API response containing a finality provider's signing info
+// for monitoring their liveness activity.
+message SigningInfoResponse {
+  string fp_btc_pk_hex = 1;
```

Comment:
every field need to have a comment otherwise `make proto-lint` won't pass (it was part of our CI but somewhere during migration we lost it 😅 )